src/lib/helpers.js

```diff
@@ -114,6 +114,40 @@ export function createHelpers({ cache, webAPI }) {
       return data
     },
 
+    async externalRefs(extRef) {
+      const obj = {}
+      extRef.forEach(ref => {
+        if (ref.type === 'his.odm.sites.SiteID') {
+          obj.siteId = ref.identifier
+        }
+        if (ref.type === 'his.odm.sites.SiteCode') {
+          obj.siteCode = ref.identifier
+        }
+        if (ref.type === 'his.odm.sites.VerticalDatum') {
+          obj.verticalDatum = ref.identifier
+        }
+        if (ref.type === 'his.odm.sites.LocalX') {
+          obj.localX = ref.identifier
+        }
+        if (ref.type === 'his.odm.sites.LocalY') {
+          obj.localY = ref.identifier
+        }
+        if (ref.type === 'his.odm.sites.PosAccuracy_m') {
+          obj.posAccuracy_m = ref.identifier
+        }
+        if (ref.type === 'his.odm.sites.State') {
+          obj.state = ref.identifier
+        }
+        if (ref.type === 'his.odm.sites.County') {
+          obj.county = ref.identifier
+        }
+        if (ref.type === 'his.odm.sites.Comments') {
+          obj.comments = ref.identifier
+        }
+      })
+      return obj
+    },
+
     // NOTE: Async since we could add slug lookup in the future
```

Comment:
This was a note to myself for a feature. Just remove the note and remove the async (and await in other places). It's just a simple way to override the org using an env var for testing.